cluster/examples/kubernetes/noobaa/noobaa-bucket-class-crd.yaml

```diff
@@ -1,14 +1,41 @@
+###############################################################################
+#
+# CRD: NooBaaBucketClass
+#
+# WAIT !!! No need to create this manually, the operator does it.
+#
+###############################################################################
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: noobaabucketclasses.noobaa.rook.io
 spec:
+  scope: Namespaced
+  group: noobaa.rook.io
```

Comment:
Is this namespaced resource required only for noobaa, or is it meant to be an addition to general bucket provisioning?